linux/jdk/redhat/src/main/packaging/temurin/22/temurin-22-jdk.spec

```diff
@@ -3,10 +3,10 @@
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 22.0.0.0.0___22.0.0.0.0+1
-#  22.0.0.0.0___1 == 22.0.0.0.0+35
-%global spec_version 22.0.0.0.0.0
+#  22.0.0.0.0___1 == 22.0.0.0.0+36
+%global spec_version 22.0.0.0.0.36
 %global spec_release 1
-%global priority 1161
+%global priority 2200
```

Comment:
Where does this number come from? They don't seem to be consistent across versions.

Reply:
To the best of my knowledge, it has to be the same for all versions of a package. I don't think we actually use it anywhere, but maybe @jiekang can offer some insight ?

Reply:
It's used for the priority value in alternatives, a Linux tool for selecting the JDK to use for a system. For example here: https://github.com/adoptium/installer/blob/master/linux/jdk/redhat/src/main/packaging/temurin/21/temurin-21-jdk.spec#L166

These comments and usage in the Fedora spec share some insight:
https://src.fedoraproject.org/rpms/java-21-openjdk/blob/rawhide/f/java-21-openjdk.spec#_339

It looks like they should mimic the jdk version and be updated when releasing new versions.